src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationTransactionalService.kt

```diff
@@ -84,6 +90,26 @@ class CalculationTransactionalService(
     }
   }
 
+  fun relevantRemandCalculation(prisonerId: String, request: RelevantRemandCalculationRequest): RelevantRemandCalculationResult {
```

Comment:
Is the point of this class, that it deals with storing data in the database? Should this function live here?